charts/refarch-templates/templates/imagestream.yml

```
@@ -2,7 +2,7 @@
 kind: ImageStream
 apiVersion: image.openshift.io/v1
 metadata:
-  name: {{ include "getFullnameImageStream" $ }}
+  name: {{ include "getFullnameImageStream" (list $.Release.Name $.Values.imagestream)  }}
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)